Makefile

```diff
@@ -9,7 +9,7 @@ ARCH := amd64
 all: local
 
 local:
-	CGO_ENABLED=0 GOOS=linux go build -o bin/faasd
+	CGO_ENABLED=0 GOOS=linux go build -ldflags $(LDFLAGS) -v -o bin/faasd
```

Comment:
Is this required for make local? It slows the build down considerably.

Reply:
the -v is not but i found it useful to see what was actually happening  and to also make the cmd easier to debug because the version would actually update as a commit

Reply:
-v is fine, but the ldflags I meant